include/tvm/tir/transform.h

```diff
@@ -492,6 +500,8 @@ TVM_DLL Pass ConvertForLoopsToSerial();
  */
 TVM_DLL Pass UnifiedStaticMemoryPlanner();
 
+TVM_DLL Pass BindParams(const std::vector<const relay::ConstantNode*>& constants);
```

Comment:
Let's consider `vector<const ConstantNode*>` vs `Array<NDArray>`. The latter option will be consistent with TVM's convention where the Pass object can be used in both C++ and Python. Would you like to briefly explain the reason why we need "ConstantNode" (which is relay-specific) instead?

Reply:
Changed to `Array<NDArray>`